libbeat/outputs/fileout/docs/fileout.asciidoc

```diff
@@ -49,12 +49,18 @@ The default value is `true`.
 The path to the directory where the generated files will be saved. This option is
 mandatory.
 
+The current epoch timestamp (in milliseconds) may be included in the path by using the `${TIME_NOW}` variable.
+For example: `/tmp/${TIME_NOW}`
```

Comment:
I find the format a inconsistent and a bit confusing given that this is the same way Beats instructs users to reference environment variables in the [docs](https://www.elastic.co/guide/en/beats/libbeat/8.12/config-file-format-env-vars.html).

The way that users specify dates and times in other parts of the configuration is like `%{+yyyy.MM.dd}`. For consistency I think we should be supporting the same format here. For example, see how the elasticsearch index can be specified in https://www.elastic.co/guide/en/beats/filebeat/master/elasticsearch-output.html#index-option-es.

This format is described in https://www.elastic.co/guide/en/beats/libbeat/8.12/config-file-format-type.html#_format_string_sprintf. And the way this is implemented is by using:

https://github.com/elastic/beats/blob/e7e6dacfca2055d14fba269253dade2d197906d8/libbeat/common/dtfmt/doc.go#L18-L54